src/test/ui/consts/const_unsafe_unreachable_ub.stderr

```diff
@@ -0,0 +1,20 @@
+error: any use of this value will cause an error
+  --> $SRC_DIR/libcore/hint.rs:LL:COL
+   |
+LL |     unsafe { intrinsics::unreachable() }
```

Comment:
The error is showing the internal code instead of the `const_unsafe_unreachable_ub.rs` source code, but I'm not sure how to make it show the source code instead. We still have the source code mentioned in the stack below though. Do you think this is a blocker?